LICENSE

```diff
@@ -19,3 +19,11 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+----
+
+SQuaLity references and may use certain test suites from external projects,
```

Comment:
perhaps you could just write something like this? When executing the script X, SQuaLity downloads the test suites of X, Y, and Z. These test suites have separate licenses:
* link and license to database 1
* link and license to database 2
* ...